x-pack/plugins/lens/public/visualizations/xy/add_layer.tsx

```diff
@@ -156,6 +208,22 @@ export function AddLayerButton({
                 },
               ],
             },
+            {
+              id: AddLayerPanelType.layerVisualizationType,
+              initialFocusedItemIndex: currentLayerVisType,
+              title: i18n.translate('xpack.lens.layerPanel.layerVisualizationType', {
+                defaultMessage: 'Layer visualization type',
```

Comment:
Can we change this text to "Select visualization type"? That will make it more consistent with the wording for the similar header when adding an annotation layer.

```suggestion
                defaultMessage: 'Select visualization type',
```

Reply:
done!